vmdb/app/controllers/application_controller.rb

```diff
@@ -966,8 +966,8 @@ def get_reports_menu(group=session[:group], tree_type="reports", mode="menu")
     rec = MiqGroup.find_by_id(group)
     user = current_user
     @sb[:grp_title] = user.admin_user? ?
-      "#{session[:customer_name]} (#{_("All %s") % ui_lookup(:models=>"MiqGroup")})" :
-      "#{session[:customer_name]} (#{_("%s") % "#{ui_lookup(:model=>"MiqGroup")}: #{user.current_group.description}"})"
+      "#{current_brand.customer_name} (#{_("All %s") % ui_lookup(:models=>"MiqGroup")})" :
+      "#{current_brand.customer_name} (#{_("%s") % "#{ui_lookup(:model=>"MiqGroup")}: #{user.current_group.description}"})"
```

Comment:
@kbrock @Fryguy @dclarizio As I mentioned, my only concern is that in Bootstrap, "brand" refers to the logo at the top left of the header (i.e., navbar-brand class) and above the login panel. 

There should be a clear distinction between that brand (which also should be part of branding) and the new "current_brand", which we have referred to as  "whitelabel" in the css. I also agree with @fryguy that "current_tenant" is more clear.